doc/esmvalcore/datafinder.rst

```diff
@@ -1,7 +1,227 @@
-.. _datafinder:
+:: _inputdata:
 
 ***********
 Data finder
 ***********
 
-Documentation of the _data_finder.py module (incl. _download.py?)
+Overview
+========
+Data discovery and retrieval is the first step in any evaluation process; ESMValTool
+uses a `semi-automated` data finding mechanism performed by the ``_data_finder.py`` module
```

Comment:
I think @bouweandela suggested not to mention private modules in the documentation.

Reply:
removed